examples/server-kitchen-sink/.storybook/preview.js

```diff
@@ -1,7 +1,7 @@
 import { addParameters, addDecorator } from '@storybook/server';
-import { withA11y } from '@storybook/addon-a11y';
+import { withLinks } from '@storybook/addon-links';
 
-addDecorator(withA11y);
+addDecorator(withLinks);
```

Comment:
I'd really like to understand why this decorator is needed for links but not for backgrounds or a11y. @tmeasday any hints?

Reply:
@jonspalmer for backgrounds I can answer that -- it doesn't do anything on the preview side; it simply reads the `parameters.backgrounds` key and uses that to drive the setting of some CSS on the manager side.

For `withA11y` I would have expected you do need to install a decorator. Perhaps the confusion is that the `a11y` preset installs the decorator already? Perhaps you've already worked this out too :)

Reply:
Yup. I mostly got there. Its interesting that the background addon doesn't support being disabled. unless you used the deprecated decorator. Maybe we should fix that sparately.

Reply:
Yeah, we could definitely check for `disabled:true` in the manager side code.

Reply:
I also don't understand why this is needed. Can we clean this up in a followup?